en/components/themes/themes-list/fluent.md

```diff
@@ -0,0 +1,19 @@
+### Switching from Material to Fluent theme
+Since version `8.2` our components support a new theme, built based on the [`Microsoft Fluent Design System`](https://www.microsoft.com/design/fluent/).  
```

Comment:
```suggestion
Since version `8.2` our components include a new theme inspired by the [`Microsoft Fluent Design System`](https://www.microsoft.com/design/fluent/).  
```